git_machete/client.py

```diff
@@ -1538,11 +1538,6 @@ def __sync_annotations_to_definition_file(self,
                 anno: str = f'PR #{pr.number}'
                 if pr.user != current_user:
                     anno += f' ({pr.user})'
-                upstream: Optional[LocalBranchShortName] = self.up_branch.get(LocalBranchShortName.of(pr.head))
-                if pr.base != upstream:
-                    warn(f'branch {bold(pr.head)} has a different base in PR #{bold(str(pr.number))} ({bold(pr.base)}) '
-                         f'than in machete file ({bold(upstream) if upstream else "<none, is a root>"})')
-                    anno += f" WRONG PR BASE or MACHETE PARENT? PR has {pr.base}"
```

Comment:
Hmmm okay but in general we want this case to be covered 🤔 it's just that we want to check for the tracking branch name first before doing the check

Reply:
Specifically, we want to extract the case like:
* PR in GitHub is from `foo` to `bar`
* locally, there's `baz` branch whose remote tracking branch is called `origin/bar`
* machete file is just `foo` under `baz`

Now if we weren't checking remote tracking branches, we'd say that `foo` has a wrong PR base or machete parent (`bar` vs `baz`)... while in fact everything's in sync.

Reply:
@PawelLipski, what do you mean by tracking branch?

Reply:
`git_machete.git_operations.GitContext.get_combined_counterpart_for_fetching_of_branch` ;)